pkg/opssight/ctl_opssight.go

```diff
@@ -59,7 +59,12 @@ type CRSpecBuilderFromCobraFlags struct {
 	ScannerPodReplicaCount                          int
 	ScannerPodImageDirectory                        string
 	PerceiverEnableImagePerceiver                   string
+	PerceiverEnableArtifactoryPerceiver             string
+	PerceiverEnableArtifactoryPerceiverDumper       string
+	PerceiverEnableQuayPerceiver                    string
 	PerceiverEnablePodPerceiver                     string
+	PerceiverArtifactoryExpose                      string
+	PerceiverQuayExpose                             string
```

Comment:
Similar question, what are we exposing for Artifactory or Quay

Reply:
For Webhooks, they send a JSON to the perceivers with Image Info which needs to be scanned